redis/src/main/scala/zio/redis/Redis.scala

```diff
@@ -16,27 +16,22 @@
 
 package zio.redis
 
+import zio._
 import zio.schema.codec.Codec
-import zio.{Has, URLayer, ZIO, ZLayer}
 
 trait Redis {
   def codec: Codec
   def executor: RedisExecutor
 }
 
-object Redis {
-
-  lazy val live: URLayer[Has[RedisExecutor] with Has[Codec], Has[Redis]] =
-    ZLayer.identity[Has[Codec]] ++ ZLayer.identity[Has[RedisExecutor]] >>> RedisService
+case class RedisLive(cc: Codec, et: RedisExecutor) extends Redis {
+  override def codec: Codec            = cc
+  override def executor: RedisExecutor = et
+}
```

Comment:
Two things:
- please make it final
- please adhere to the [multi-unit files naming convention](https://docs.scala-lang.org/style/files.html)